tests/core/tests.rs

```diff
@@ -118,4 +118,13 @@ mod tests {
         assert_eq!(0u64, U64::from(0u64));
         assert_eq!(0i64, I64::from(0i64));
     }
+
+    #[test]
+    fn signed_ints() {
+        // the + 1 is due to a bug in cbor_event that doesn't cover the entire range of i64
```

Comment:
Probably best to create an issue about this in the cbor_event repo and then link to the issue in this comment